sssd_test_framework/roles/ad.py

```diff
@@ -1490,4 +1545,216 @@ def __get_info(
         return info
 
 
+class GPO(BaseObject[ADHost, AD]):
+    """
+    Group policy object management.
+    """
+
+    def __init__(self, role: AD, name: str) -> None:
+        """
+        :param role: AD host object.
+        :type role:  ADHost
+        :param name: GPO name, defaults to 'Domain Test Policy'
+        :type name: str, optional
+        """
+        super().__init__(role)
+
+        self.name: str = name
+        """Group policy display name."""
+
+        self._search_base: str = f"cn=policies,cn=system,{self.role.host.naming_context}"
+        """Group policy search base."""
+
+        self._dn: str = self.get("DistinguishedName")
+        """Group policy dn."""
+
+        self._cn: str = self.get("CN")
+        """Group policy cn."""
+
+    def get(self, key: str) -> str:
```

Comment:
What is the use case for this method? Why does it allow to retrieve only a single key instead of set of attributes?

Reply:
Fewer lines of code and a simple way to get the value was the goal. 

I tried using parse_attrs, right now on the multiline values, the string isn't stripped correctly, so the values look like "...dc=ad,dc                                                     =test", which doesn't work injecting the values into powershell code. I'm going to fix that later.